ecs-agent/api/resource/resource_type.go

```diff
@@ -14,6 +14,12 @@
 package resource
 
 const (
-	EphemeralStorage    = "EphemeralStorage"
+	// EphemeralStorage is one of the resource types in the properties list of the attachment payload message for the
+	// ephemeral storage.
+	EphemeralStorage = "EphemeralStorage"
+	// ElasticBlockStorage is one of the resource types in the properties list of the attachment payload message for the
+	// EBS volume on firecracker.
 	ElasticBlockStorage = "ElasticBlockStorage"
+	// AmazonElasticBlockStorage is one of the attachment types in the attachment payload message for EBS attach tasks on warmpool.
```

Comment:
> AmazonElasticBlockStorage is one of the attachment types in the attachment payload message for EBS attach tasks on warmpool.

nit: this might be true for Fargate but I'm not sure if this applies to EC2. Perhaps we can make this more general by saying "AmazonElasticBlockStorage is one of the....for EBS attach task" and remove the last bit about warmpool. Thoughts on this?

Reply:
Yes, my bad. Updating it.

Reply:
thanks!

Reply:
+1